near-sdk/src/environment/env.rs

```diff
@@ -690,7 +690,21 @@ pub fn panic(message: &[u8]) -> ! {
     }
     unreachable!()
 }
+/// Logs the string message message. This message is stored on chain.
+pub fn log_str(message: &str) {
+    #[cfg(all(debug_assertions, not(target_arch = "wasm32")))]
+    println!("{}", message);
```

Comment:
Nice! Should this be `eprintln`?

Reply:
ah, good point, this probably should, along with the other, so it's matched with the test output